src/aiori-HDF5.c

```diff
@@ -19,17 +19,21 @@
 #include <stdio.h>              /* only for fprintf() */
 #include <stdlib.h>
 #include <sys/stat.h>
+#include <unistd.h>
+#include <dlfcn.h>
 /* HDF5 routines here still use the old 1.6 style.  Nothing wrong with that but
  * save users the trouble of  passing this flag through configure */
-#define H5_USE_16_API
+//#define H5_USE_16_API
 #include <hdf5.h>
 #include <mpi.h>
+#include <daos.h>
 
 #include "aiori.h"              /* abstract IOR interface */
 #include "utilities.h"
 #include "iordef.h"
 
 #define NUM_DIMS 1              /* number of dimensions to data set */
+#define RANK_LIST "1:2:3:4:5:6:7:8:9:10:11:12:13:14:15:16" /* Rank list for excluding a server */
```

Comment:
sorry i don't get what this is.. what if we have less than that?
what if we have much more?

Reply:
I fixed it.